### DISCUSSION TITLE
How about let's build a device-service store?

### DISCUSSION THREAD

**Discussion by wanhua-leo** [edgexfoundry/community#231] [Ideas]
I know there is already a serial of device-services, but each one has some issues when i use it in my scene, so maybe we can build a store to share more device-services.

**Comment by jiekechoo** [comment]
@wanhua-leo , you could find all device services in [github.com/edgexfoundry](https://github.com/orgs/edgexfoundry/repositories), like `device-***`. The community has shared a [docker-compose ](https://github.com/edgexfoundry/edgex-compose) repo for testing, and released the [documentation site](https://docs.edgexfoundry.org) for users.

If you have any issue, welcome to submit it in spectific repo.

**Reply by wanhua-leo** [reply]
jieke, thanks for your information, now each protocol has only one device-service, I feel it will be better we can have more choices and has an easier way to use it.

**Reply by jiekechoo** [reply]
What way? Please let me know.

**Reply by wanhua-leo** [reply]
the Modbus protocols we work with are various variations in them, such as, big-endian or little-endian, bitwise value, the number of multiple ​​getting at one time, etc.

**Reply by jiekechoo** [reply]
And what's your point? Tell us more details, please.

**Comment by cloudxxx8** [comment]
@wanhua-leo you can share your device services on DockerHub.
That's the store and deployment approach we are currently using.
